packages/apps/spaces/hooks/useFileData/useFileData.ts

```diff
@@ -17,7 +17,7 @@ export const useFileData = ({ addFileToTextContent }: Props): Result => {
 
           if (dataUrl) {
             addFileToTextContent(
-              `<img width="400" src='${dataUrl}' alt='${id}'>`,
+              `<img width="400" src='${dataUrl}' alt="" id='${id}'>`,
```

Comment:
i think this is going to break the load of the pictures. please check the NoteTimelineItem.tsx at line 71